README.md

```diff
@@ -139,57 +139,63 @@ issue and ask.
-  tracking issue number and title is the name of the proposal. As a pull request
-  the RFC will receive design feedback from the core team and the larger
-  community, and the author should be prepared to make revisions in response.
+   tracking issue number and title is the name of the proposal. As a pull
+   request the RFC will receive design feedback from the core team and the
+   larger community, and the author should be prepared to make revisions in
+   response.
 6. Update the tracking issue with a link to the RFC PR.
-7. **Advertise** your RFC amongst stakeholders via social channels (e.g. twitter)
-  and your team. Build consensus and integrate feedback. RFCs that have broad support 
-  are much more likely to make progress than those that don't receive any comments.
+7. **Advertise** your RFC amongst stakeholders via social channels (e.g.
+   twitter) and your team. Build consensus and integrate feedback. RFCs that
+   have broad support are much more likely to make progress than those that
+   don't receive any comments.
 8. Eventually, the team will decide whether the RFC is a candidate for inclusion
-  in CDK/JSII.
-9. RFCs that are candidates for inclusion in CDK/JSII will enter a "**final comment
-  period**" lasting 3 calendar days. The beginning of this period will be signaled
-  by a team member adding a comment and label on the RFCs pull request.
+   in CDK/jsii.
+9. RFCs that are candidates for inclusion in CDK/jsii will enter a "**final comment
+   period**" lasting 3 calendar days. The beginning of this period will be signaled
+   by a team member adding a comment and label on the RFCs pull request.
 10. An RFC can be modified based upon feedback from the team and community.
-  Significant modifications may trigger a new final comment period.
-11. An RFC may be **rejected** by the team after public discussion has settled and
-  comments have been made summarizing the rationale for rejection. A member of
-  the team will then close the PR and issue.
-12. An RFC may be accepted at the close of its final comment period. A team member
-  will merge the RFCs associated pull request, at which point the RFC will
-  become 'approved'.
-13. At some point, someone will pick up the RFC for implementation. For major features
-  this usually requires devising a detailed implementation plan. To that end, submit
-  an **additional PR** on the RFC doc that fills in the "Implementation Plan" section.
+    Significant modifications may trigger a new final comment period. An RFC can
+    also be modified after it has been merged and approved, in which case a new
+    PR will be submitted with the modification, like any other code.
+11. An RFC may be **rejected** by the team after public discussion has settled
+    and comments have been made summarizing the rationale for rejection. A
+    member of the team will then close the PR and issue.
+12. An RFC may be accepted at the close of its final comment period. A team
```

Comment:
```suggestion
12. An RFC may be **accepted** at the close of its final comment period. A team
```

Might be worth bolding the "accepted" since you bolded the "rejected".